portfolio/src/main/java/Server2.java

```diff
@@ -0,0 +1,16 @@
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;/** Servlet that responds with a basic header text. */
+
+@WebServlet("/head")
```

Comment:
nit: what does head mean? An endpoint that tells you where you're going when you call it makes it easier to read the code!